### PR TITLE
[Doc] add dump_query.md

### PR DESCRIPTION
**Pull request by Seaven**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] I have added user document for my new feature or new function

### CODE REVIEW

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

File: docs/faq/Dump_query.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
+# QueryDump Interface Documentation
```
Comment (imay) [inline docs/faq/Dump_query.md:1]: 1. needs to add this document in TOC.md then users can see this document in the content list

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Comment by wanpengfei-git** [comment]
starrocks_be_unittest succeeded.

**Review by imay — APPROVED** [review]

**Review by kangkaisen — APPROVED** [review]

**Pull request merged**